config/poro.yml

```diff
@@ -23,8 +23,8 @@ entries:
 - prefix: /about/
   replacement: http://www.ontobee.org/browser/rdf.php?o=PORO&iri=http://purl.obolibrary.org/obo/
```

Comment:
This replacement still points to Ontobee, so the test that points to EBI fails (and with #544 we can now see the failure).

The `term_browser` field is specifically for the `http://purl.obolibrary.org/obo/ABC_123` case. There's no special handling of `/about/`.